client/src/components/ResourceManager.jsx

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import React, { useEffect, useState } from 'react';
+import { Menu } from 'antd';
+import { DownOutlined, EditOutlined, CloseOutlined } from '@ant-design/icons';
+
+import { getCategories } from '../utils/api';
+
+import '../css/ResourceManager.css';
+
+const ResourceManager = () => {
+  const { SubMenu } = Menu;
+  const [categories, setCategories] = useState<{ [string]: Array<string> }>({});
+
+  useEffect(() => {
+    const fetchCategories = async () => {
+      const res = await getCategories();
+      const newCategories = {};
+      if (res != null) {
+        res.result.forEach((c) => {
+          newCategories[c.name] = c.subcategories;
+        });
+      }
+      setCategories(newCategories);
+    };
+
+    fetchCategories();
+  }, []);
+
+  return (
+    <Menu style={{ width: 280 }} mode="inline" expandIcon={<div />}>
```

Comment:
What's the exapandIcon for?

Reply:
The expandIcon (located on the right) is traditionally a dropdown arrow, although we wanted to disable that righthand arrow to place the edit/close icons in that spot. I made that happen by passing in an empty div.